packages/std/src/addresses.rs

```diff
@@ -170,9 +194,20 @@ mod test {
     }
 
     #[test]
-    fn human_addr_implements_partial_eq() {
-        let human_addr = HumanAddr::from("cos934gh9034hg04g0h134");
-        assert_eq!(&human_addr, "cos934gh9034hg04g0h134");
```

Comment:
Before this PR, we could do

`assert_eq!(&human_addr, "cos934gh9034hg04g0h134");`

but not

`assert_eq!("cos934gh9034hg04g0h134", &human_addr);`